content/livesync/quickstart.textile

```diff
@@ -1,7 +1,240 @@
+You can have the freedom to choose the Postgres database you wish to use, but for the purposes of the quickstart, this step will take you through setting up a database with "Neon":https://neon.tech, a serverless platform for Postgres databases. Once created, copy the Postgres credentials to be used in the next step.
+
+h3(#step-3). Update environment variables
+
+Set up environment variables by copying them from the template:
+
+```[sh]
+ cp env.example env.local
+```
+
+Update your @env.local@ file with the following:
+
+```
+POSTGRES_URL="<YOUR_FULL_POSTGRES_URL>"
+POSTGRES_URL_NON_POOLING="<YOUR_FULL_POSTGRES_URL>"
+SESSION_SECRET=somesecret
+NEXT_PUBLIC_ABLY_API_KEY=<YOUR_ABLY_API_KEY>
+```
+
+- Replace the @POSTGRES_URL@ and @POSTGRES_URL_NON_POOLING@ values with your Neon (or alternative) postgres URL.
+- Replace the @POSTGRES_USER@, @POSTGRES_HOST@, @POSTGRES_PASSWORD@, and @POSTGRES_DATABASE_NAME@ values with those relevant to your database.
+- Replace @<YOUR_ABLY_API_KEY>@ with your Ably API key.
+
+Export the environment variables in your shell session:
+
+```[sh]
+export $(grep -s -v "^#" env.local | xargs) # export environment variables
+```
+
+h3(#step-4). Configure hosted Database Connector
+
+The Database Connector watches changes in your database table using the transactional "outbox pattern":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html and broadcast those updates over Ably Channels. There are two Database Connectors you can use, either the "Ably hosted":/livesync/database-connector#hosted-with-ably or "self hosted":/livesync/database-connector#self-hosted, for this quickstart we'll use the Ably hosted Database Connector. In the "Ably dashboard":https://ably.com/dashboard, choose the project you wish to use for this quickstart and open the @Integrations@ tab. Update the following options available to you:
+
+|_. Option |_. Description |
+| URL | Your Neon Postgres database full URL |
+| Outbox table schema | Schema for the "outbox table":/livesync/outbox-nodes-tables#schema. Use "public" for this quickstart. |
+| Outbox table name | Name for the "outbox table":/livesync/outbox-nodes-tables#outbox-table. In this quickstart it is "outbox" |
+| Nodes table schema | Schema for the "nodes table":/livesync/outbox-nodes-tables#schema. Use "public" for this quickstart. |
+| Nodes table name | Name for the "nodes table":/livesync/outbox-nodes-tables#nodes-table. In this quickstart it is "nodes" |
+| SSL mode | Determines the level of protection provided by the SSL connection. Use the default value (@prefer@) for this quickstart. |
+| SSL root certificate | Leave this empty for the quickstart. |
+| Primary site | The primary data center in which to run the integration rule. |
+
+h3(#step-5). Install dependencies
+
+Install the required dependencies for the web application, including your Models SDK. Then create the necessary database tables and create some seeded data:
+
+```[sh]
```

Comment:
Not sure as a reader I understand based on the description above what the run db command would do - can we please clarify?